util/init.lua

```diff
@@ -111,13 +111,13 @@ end
 -- {{{ Dynamic tagging
 
 -- Add a new tag
-function util.add_tag()
+function util.add_tag(layout)
     awful.prompt.run {
         prompt       = "New tag name: ",
         textbox      = awful.screen.focused().mypromptbox.widget,
         exe_callback = function(name)
             if not name or #name == 0 then return end
-            awful.tag.add(name, { screen = awful.screen.focused() }):view_only()
+            awful.tag.add(name, { screen = awful.screen.focused(), layout = layout }):view_only()
```

Comment:
You should add `layout = layout or awful.layout.layouts[0]` to catch `nil` inputs.

Plus, a hint in the wiki too.

You can `git commit` the addition,  `git rebase -i HEAD~2` to squash it and `git push --force` to update this PR.